crates/codegen/runtime/cargo/src/runtime/parse_error.rs

```diff
@@ -43,52 +39,41 @@ impl ParseError {
     }
 }
 
-pub(crate) fn render_error_report(
-    error: &ParseError,
-    source_id: &str,
-    source: &str,
-    with_color: bool,
-) -> String {
-    use ariadne::{Color, Config, Label, Report, ReportKind, Source};
-
-    let kind = ReportKind::Error;
-    let color = if with_color { Color::Red } else { Color::Unset };
-
-    let tokens_that_would_have_allowed_more_progress =
-        error.tokens_that_would_have_allowed_more_progress();
-    let message = if tokens_that_would_have_allowed_more_progress.is_empty() {
-        "Expected end of file.".to_string()
-    } else {
-        format!(
-            "Expected {expectations}.",
-            expectations = tokens_that_would_have_allowed_more_progress.join(" or ")
-        )
-    };
-
-    if source.is_empty() {
-        return format!("{kind}: {message}\n   ─[{source_id}:0:0]");
+impl Error for ParseError {}
```

Comment:
The `Error` impl is just for completeness' sake here but will probably be unused because we use richer result for the parsing, i.e. `ParseOutput`, so will probably not be used. Let me know if you want to keep it or not.

Reply:
I agree with your assumption here. Generally speaking, if there is no use/need for it, I suggest not paying the cost of maintaining it.

Reply:
as you can see, there isn't much cost really in `impl Error for ParseError {}` but I'm happy to prune it until we find a use case for it :+1: